tests/routers/openml/task_type_test.py

```
@@ -38,3 +39,29 @@ def test_get_task_type_unknown(py_api: TestClient) -> None:
     response = py_api.get("/tasktype/1000")
     assert response.status_code == HTTPStatus.PRECONDITION_FAILED
     assert response.json() == {"detail": {"code": "241", "message": "Unknown task type."}}
+
+
+def test_get_task_type_invalid_constraint(
+    py_api: TestClient,
+    expdb_test: Connection,
+) -> None:
+    expdb_test.execute(
+        text(
```

Ruff (0.14.5)

Undefined name `text`

(F821)